react-frontend/package.json

```diff
@@ -14,6 +14,7 @@
     "react-apollo": "^3.1.5",
     "react-dom": "^16.13.1",
     "react-flexbox-grid": "^2.1.2",
+    "react-head": "^3.3.0",
```

Comment:
Is this used anywhere?  I could not find it declared anywhere.

Reply:
Might need for the icon import, will update this when I get it figured out

Reply:
Don't need it, removing